pkg/disk/utils.go

```diff
@@ -1102,6 +1103,19 @@ func getDiskVolumeOptions(req *csi.CreateVolumeRequest) (*diskVolumeArgs, error)
 		diskVolArgs.ResourceGroupID = ""
 	}
 
+	// allowIncrease
+	value, ok = volOptions["allowIncrease"]
```

Comment:
@fredkan Thanks for the review.

| 1, maybe "volumeSizeAutoAvailable" is better?
I don't have any preference to the naming of the flag. If you prefer this then that's fine. I have updated the flag.
| 2. this should be true by default, Otherwise the default storageclass will not work.
As @jsafrane has previously mentioned and from our conversation with Bo Teng, we have decided to allow this flag on a newly created storage class. This will also preserve the current behavior when creating volumes for customers who do not want the confusion of having larger disks than the PVC requested size.